src/test/configuration_snippet.rs

```diff
@@ -251,6 +255,7 @@ fn configuration_snippet_tests() {
     let blocks = get_code_blocks();
     let failures = blocks
         .iter()
+        .filter(|block| !block.fmt_skip())
```

Comment:
Was this change related/necessary for the fix? It seems reasonable enough but wasn't sure if it was just an add on

Reply:
Yeah it was needed. The tests fail otherwise. The original PR (4297) ran into a similar issue, and [here's how they chose to fix it](https://github.com/rust-lang/rustfmt/pull/4297/commits/2127bdf594eb6513bb75ccd4d242934e9c9657fd). The test code has changed a bit since (4297) was opened, but the comment in the commit is insightful as to what's happening and why we need to skip checking these code blocks.

Let me know if you have another suggestions for fixing the test.